docs/getting-started/public-cloud-server.md

```
@@ -1,38 +1,145 @@
+### How to Use 🛠️
+
+1. **Activate Cloud Mode**  
+   Launch app → Toggle "Cloud Server"
+
+2. **Go Online**  
+   Tap the "Offline" button → Becomes "Online"
+   
+3. **Get Credentials**  
+   They will be generated by the server.
+   <center>
+      <img src="/assets/cloud-server.png"/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)